src/assets/animations/pulse.js

```diff
@@ -0,0 +1,20 @@
+import { keyframes } from 'styled-components';
+
+const Pulse = keyframes`
+    0% {
+        transform: scale(0.95);
+        box-shadow: 0 0 0 0 rgba(0, 0, 0, 0.5);
+    }
+
+    50% {
+        transform: scale(1);
+        box-shadow: 0 0 0 10px rgba(0, 0, 0, 0);
+    }
+
+    100% {
+        transform: scale(0.95);
+        box-shadow: 0 0 0 0 rgba(0, 0, 0, 0);
+    }
+`;
+
+export default Pulse;
```

Comment:
I like that you extracted this. It is reusable on all your components. Good work.